scripts/SHI/scpSHI_Jetz_complete.yml

```diff
@@ -0,0 +1,93 @@
+script: scpSHI_Jetz_complete.R
+description: 
+  This script loads the data needed to calculate the area and connectivity scores 
+  to calculate the Species Habitat Index (SHI) using Global Forest Watch data to measure 
+  changes in the habitat of the species
+inputs:
+  species:
+    label: species
+    description: Scientific name of the species
+    type: text
+    example: Saguinus oedipus
+  ref_system:
+    label: Projection
+    description: Reference system for the area of interest
+    type: int
+    example: 3116
+  elev_margin:
+    label: Elevation buffer
+    description: Magnitude of buffer to add to the elevation ranges of the species (usualy in meters), if no buffer then 0
+    type: int
+    example: 0
+  forest_threshold:
+    label: Tree cover percentage 
+    description: 
+      Percentage of tree cover of reference for the Global Forest Watch data according to the tolerance of the species. 
+      This value is given according to suitable habitat of the species. If not available, use Map of Life Values 
+      (e.g. [https://mol.org/species/range/Saguinus_oedipus]) 
+    type: int
+    example: 50
+  country_code:
+    label: Country code
+    description: Three letter code (ISO_3166-1_alpha-3) for the country where the analysis will be done.
+    type: text
+    example: COL
+  region:
+    label: Region
+    description: Name of the region according to the codification from the GADM data [https://gadm.org/metadata.html] for the variable NAME_1
+    type: text
+    example: ""
+  spat_res:
+    label: Spatial resolution
+    description: Spatial resolution for the analysis, in meters
+    type: int
+    example: 1000
+  expert_source:
+    label: Expert range source
+    description: Source of the expert range map
+    type: options
+    options:
+      - MOL
+      - IUCN
+      - QC
+    example: IUCN
+  t_0:
+    label: Initial time
+    description: Year where the analysis should start
+    type: int
+    example: 2000
+  t_n:
+    label: Final time
+    description: Year where the analysis should end (it should be later than Initial time).
+    type: int
+    example: 2020
+  time_step:
+    label: Time step
+    description:
+      Time range between each time step. It can be one to get change every year or two if the change 
+      is going to be evaluated every two years and so on.
```

Comment:
Time interval to perform the analysis between t_0 and t_n. For instance, ervery 2 years.

analysis is  calculating land cover area and connectivity??